Makefile

```diff
@@ -1,6 +1,9 @@
 # Run `make` (no arguments) to get a short description of what is available
 # within this `Makefile`.
-
+# Our user and group id used as default container user:group id. Avoid permissions pollution
+UID := $(shell id -u)
+GID := $(shell id -g)
```

Comment:
I think this is breaking the `make start` command in MacOS as usually users are in the `staff` group gid 20 in Mac which is already taken in the Linux container so will throw an error.

I would suggest as an easy fix setting a default UID and GID to 1000.